src/Nova/Components/WorkspaceServicesFields/WorkspaceServicesFields.php

```diff
@@ -3,6 +3,7 @@
 namespace Webid\Octools\Nova\Components\WorkspaceServicesFields;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Crypt;
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**🎯 suggestion:** ‏j'ai pas l'impression que ce soit utilisé. A supprimer du coup.